bionic/flow.py

```diff
@@ -959,6 +959,23 @@ def get(self, name, collection=None, fmt=None, mode=object):
         else:
             raise ValueError("Unrecognized collection type %r" % collection)
 
+    def get_docstring_for_entity(self, name):
+        """
+        Returns the docstring for function underlying entity ``name``.  Only supported for
+        FunctionProviders (not ValueProviders).  If no docstring is present, returns ``None``.
+
+        Parameters
+        ----------
+
+        name: String
+            The name of an entity.
+        """
+        state = self._state
+        provider = state.providers_by_name[name]
```

Comment:
If you use `state.get_provider(name)`, it will throw a more helpful exception when the entity name doesn't exist.